src/main/java/com/google/sps/data/EmailSender.java

```diff
@@ -81,10 +39,5 @@ public static String fileContentToString(String filePath) throws IOException {
   // Ex. str = "You will be mock interviewing {{interviewee_full_name}} on {{formatted_date}}."
   // toReplace = { ("{{interviewee_full_name}}","Tess"), ("{{formatted_date}}", "June 6, 2022") }
   // Returned: "You will be mock interviewing Tess on June 6, 2022."
-  public static String replaceAllPairs(HashMap<String, String> toReplace, String str) {
-    for (Map.Entry<String, String> entry : toReplace.entrySet()) {
-      str = str.replace(entry.getKey(), entry.getValue());
-    }
-    return str;
-  }
+  public String replaceAllPairs(HashMap<String, String> toReplace, String str);
```

Comment:
This also shouldn't be part of the interface. Conceptually, an email sender should send emails.

Reply:
Done

Reply:
Done